src/components/scene/xr-mode-ui.js

```diff
@@ -120,8 +120,6 @@ module.exports.Component = registerComponent('xr-mode-ui', {
 
     this.orientationModalEl = createOrientationModal(this.onModalClick);
     sceneEl.appendChild(this.orientationModalEl);
-
-    this.updateEnterInterfaces();
```

Comment:
This caused `updateEnterInterfaces` to be called multiple times.

Reply:
we can restore this